### PR TITLE
fix(script): fix parse issue in compare_costs.py and wrong cost in genesis

### PR DESCRIPTION
**Pull request by ailisp**
The parse didn't handle nested same name cost properly: 
```
transaction_costs.action_creation_config.function_call_cost_per_byte
transaction_costs.action_creation_config.add_key_cost.function_call_cost_per_byte
```
etc. was only displayed once, but they're two different costs, fix by replace with json. 
Also the cost start in https://github.com/nearprotocol/nearcore/commit/f024a28d9b0f1b2238f04d288ddebc8ac12ff4d1 seems wrong: I get
a. current genesis.json in master (cost calculated from f024ad8)
b. I rerun cost estimator on master, get genesis.json runtime_configs (number is in this PR's genesis.json)
c. my rerun cost estimator in f024ad8: https://transfer.sh/ESS0J/runtime_config_f024a28d.json

Above, b. and c. number is very close, but a is very different from c.

Test Plan
---------
Run compare-costs.py, see transaction_costs.action_creation_config.function_call_cost_per_byte and transaction_costs.action_creation_config.add_key_cost.function_call_cost_per_byte are both occurred

### CODE REVIEW

**Comment by ailisp** [comment]
Hmm i misread, function_call_cost_per_byte is correct but add_key_cost.function_call_cost_per_byte it's still problematic, 10000x different, will bin search which cause that wrong

**Comment by ailisp** [comment]
Figured out :joy: previous add_key_cost.function_call_cost_per_byte was wrong. In my run it's close to @olonho  result (latest master). Checkout to https://github.com/nearprotocol/nearcore/tree/fast-funcall-per-byte-cost  (measure only add_key_cost.function_call_cost_per_byte)
or https://github.com/nearprotocol/nearcore/commit/f024a28d9b0f1b2238f04d288ddebc8ac12ff4d1 (measure all metrics)
to see. Can any one also try and confirm result

**Comment by ailisp** [comment]
Retry two times on f024a28 and one time on the commit before f024a28, it's giving similar result to current master result (and @olonho 's result), so mostly sure f024a28 genesis is wrong, I pushed new calculated genesis.json from this branch (equivalent to current master), will update migration script and init params in rust source tomorrow.

File: runtime/runtime-params-estimator/src/runtime_fees_generator.rs [hunk 1]
```diff
@@ -92,7 +92,10 @@ impl RuntimeFeesGenerator {
         res.insert(
             ReceiptFees::ActionAddFunctionAccessKeyPerByte,
             // These are 1k methods each 10bytes long.
-            Ratio::new(self.aggregated[&Metric::ActionAddFunctionAccessKey1000Methods].upper(), 10),
+            Ratio::new(
+                self.aggregated[&Metric::ActionAddFunctionAccessKey1000Methods].upper(),
+                10 * 1000,
```
Comment (olonho) [inline runtime/runtime-params-estimator/src/runtime_fees_generator.rs:97]: +1

**Pull request merged**